crac-plugin/src/main/java/io/micronaut/gradle/crac/MicronautCRaCPlugin.java

```diff
@@ -203,19 +203,16 @@ private CheckpointTasksOfNote configureCheckpointDockerBuild(Project project,
             task.getContainerId().set(start.flatMap(DockerExistingContainer::getContainerId));
             task.getSink().fileValue(checkpointFile);
             // Do not use lambda or it's not compatible with Gradle's config cache
-            task.doLast(new Action<>() {
-                @Override
-                public void execute(Task t) {
-                    List<String> lines;
-                    try {
-                        lines = Files.readAllLines(checkpointFile.toPath());
-                    } catch (IOException e) {
-                        throw new GradleException("Checkpoint container failed");
-                    }
-                    lines.forEach(task.getLogger()::lifecycle);
-                    if (lines.stream().noneMatch(l -> l.contains("Snapshotting complete"))) {
-                        throw new GradleException("Checkpoint container failed");
-                    }
+            task.doLast(t -> {
```

Comment:
Same here, we need to use anonymous classes I think.

Reply:
fixed